bentoml/saved_bundle/config.py

```diff
@@ -99,21 +100,20 @@ def load(cls, filepath):
         ver = str(conf["version"])
 
         if ver != BENTOML_VERSION:
-            msg = (
+            msg_mismatch = (
                 "Saved BentoService bundle version mismatch: loading BentoService "
```

Comment:
More detailed raise message or keep consistency?

Reply:
I think keeping it consistent is fine